widgets/contact/contact.php

```diff
@@ -1075,6 +1075,21 @@ function contact_form_action( $instance, $storage_hash ) {
 					);
 					break;
 
+				case 'tel':
+					// Somewhat arbitrary basic phone number validation, checking for at least 3 digits, ignoring all
+					// non-digit characters. Apparently, the lower limit for phone numbers is 3. See
+					// https://github.com/siteorigin/so-widgets-bundle/issues/958#issuecomment-573139753
+					$digits = preg_replace( '/\D/', '', $value );
+					if ( strlen($digits) < 3 ) {
+						$errors[ $field_name ] = __( 'Invalid phone number. It should contain at least four digits.', 'so-widgets-bundle' );
```

Comment:
Should we perhaps keep the error message a bit more generic? 

_Invalid phone number._

And just leave it at that?

Reply:
IMO, I don't think it's very good UX to tell the user their input is invalid without telling them how to fix it.